DOC-1317 - RS: CLI Utitilites Reorg rladmin help and rladmin info commands (#1887)
RedisLabs/redislabs-docs: content/rs/references/cli-utilities/rladmin/help.md

---
Title: rladmin help
linkTitle: help
description: List all options and parameters associated with the rladmin command.
weight: $weight
alwaysopen: false
toc: "true"
headerRange: "[1-2]"
categories: ["RS"]
aliases:
---

`rladmin help` lists all options and parameters associated with the rladmin command.

``` sh
rladmin help [command]
```

### Parameters

| Parameter | Description |
|-----------|-------------|

> | command   | rladmind command to display help |

Reviewer: ```suggestion
|  command   |  Display help for this rladmin command (optional)  |
```

### Returns

> Returns a list of all the options and parameters associated with the rladmin command.
> 
> If `[command]` is specified, returns a list of all the options and parameters associated with that rladmin command.

Reviewer: ```suggestion
Returns a list of available `rladmin` commands.

If `[command]` is specified, returns a list of all the options and parameters for that `rladmin` command.
```
